Doc/c-api/sys.rst

```diff
@@ -347,12 +347,14 @@ accessible to C code.  They all work with the current interpreter thread's
    :class:`Exception` (other errors will not be silenced).
 
    .. c:namespace:: NULL
-   .. c:type:: int (*Py_AuditHookFunction) (const char *, PyObject *, void *)
-
-   The hook function is of type :c:expr:`int (*)(const char *event, PyObject
-   *args, void *userData)`, where *args* is guaranteed to be a
-   :c:type:`PyTupleObject`. The hook function is always called with the GIL
-   held by the Python interpreter that raised the event.
+   .. c:type:: int (*Py_AuditHookFunction) (const char *event, PyObject *args, void *userData)
+
+      The type of the hook function.
+      *event* is (the event argument passed to PySys_Audit() or PySys_AuditTuple()),
+      *args* is guaranteed to be a :c:type:`PyTupleObject`. The hook
+      function is always called with the GIL held by the Python
+      interpreter that raised the event.
+      *userData* is the argument passed to PySys_AddAuditHook().
```

Comment:
Should not it be immediately after describing args and before the statement about GIL?

Reply:
Added another edit. I went back and forth about the placement of the `Py_AuditHookFunction` description with respect to `PySys_AddAuditHook`, finally placing it after.